src/phoenix/trace/schemas.py

```diff
@@ -12,6 +12,10 @@ class SpanStatusCode(Enum):
     OK = "OK"
     ERROR = "ERROR"
 
+    @classmethod
+    def _missing_(cls, v: Any) -> Optional["SpanStatusCode"]:
+        return None if v else cls.UNSET
+
```

Comment:
In case the status code is missing, i.e. `span.get("status_code") is None`, it will default to `UNSET` when we do `SpanStatus(span.get("status_code"))`. It makes the code shorter. But it will still error out if we do `SpanStatus("UNK")` because "UNK" is not valid.

Reply:
Doesn't this need to be a dunder?

Reply:
no dunder, just [`_missing_`](https://docs.python.org/3/library/enum.html#enum.Enum._missing_)